src/server/system_services/stats_aggregator.js

```diff
@@ -1246,6 +1246,14 @@ async function update_nsfs_stats(req) {
     if (_nsfs_counters.fs_workers_stats) _update_fs_stats(_nsfs_counters.fs_workers_stats);
 }
 
+async function standalon_update_nsfs_stats(nsfs_stats) {
+    dbg.log1(`standalon_update_nsfs_stats. nsfs_stats =`, nsfs_stats);
+    const _nsfs_counters = nsfs_stats || {};
```

Comment:
can this line be deleted if you will add default {} on the signature? 
async function standalon_update_nsfs_stats(nsfs_stats = {}) {... }